examples/basic.html

```diff
@@ -19,6 +19,10 @@
         github: "https://github.com/w3c/some-API/",
         testSuiteURI: "https://w3c-test.org/some-API/",
         implementationReportURI: "https://w3c.github.io/test-results/some-API",
+        caniuse: {
```

Comment:
Best to revert this. Tho, of course change it for your own local testing (it's also what I do!) :)